projects/docs/docs/guides/third-party-dependencies.md

```diff
@@ -134,3 +134,14 @@ let project = Project(
     ]
 )
 ```
+
+## Swift Package Manager mapping details
```

Comment:
Docs under `guides` are for users and therefore they don't expect to see internal details in the content. I suggest hiding those from the section title and the content:

```md
## Merging Swift Packages into your project's graph

When Swift Packages are integrated into your project's graph, there are some heuristics in place to ensure the resulting graph is valid. 

First, Tuist defaults to using `iOS` as a platform when there's more than one platform defined in the `Dependencies.swift` and the package's manifest file. This is currently a limitation of Tuist [because it does not support multi-platform targets](https://github.com/tuist/tuist/issues/397).

....

```

Reply:
✅